src/main/java/com/pace/hibernate/model/Student.java

```diff
@@ -0,0 +1,27 @@
+package com.pace.hibernate.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "students")
+public class Student {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+    @Column(name = "name")
+    private String name;
+    @Column(name = "email")
+    private String email;
+    @Column(name = "address")
+    private String address;
+    @OneToMany
+    private School school;
+}

```

Comment:
@Column(name = "column_name", unique = true/false, length = ?,nullable = true/false )
try to add these as well